application_handlers_test.go

```diff
@@ -618,6 +658,30 @@ func TestApplicationEdit(t *testing.T) {
 		t.Errorf("Expected: %v Obtained: %v", emailNotificationInfo, notificationProducer.EmailNotificationInfo)
 	}
 
+	application, _ = applicationDao.GetById(&applicationID)
+	if application.AvailabilityStatus == previousApplicationStatus {
+		t.Errorf("Invalid email notification data:")
+	}
+
+	src, err = sourceDao.GetById(&application.SourceID)
+	if err != nil {
+		t.Error(err)
+	}
+
+	if src.AvailabilityStatus == previousSourceStatus {
```

Comment:
in this condition, the `src.AvailabilityStatus` has value "available" and `previousSourceStatus` has value "", so we check that `"available" == "" `... but in this case we know that expected is value "available" ... from my perspective it is maybe better to have here something like ... what you think?

`if src.AvailabilityStatus != "available" {`

Reply:
yes make sense but here we are testing whether status was changed and this condition cover only destination value after this update operation - I would leave as it is if you are ok with it anyway good insight